src/DateUtils.ts

```diff
@@ -136,6 +136,18 @@ export function formatCallTime(delta: Date): string {
     return output;
 }
 
+export function formatSeconds(inSeconds: number): string {
```

Comment:
We have a `<Clock />` component which does this for you (might need light edits to support hours though)

Reply:
I am using that component already :P See [`49defd7` (#6700)](https://github.com/matrix-org/matrix-react-sdk/pull/6700/commits/49defd7b5a52087227ecb21834ddd098dababc2c) and [`c6400c8` (#6700)](https://github.com/matrix-org/matrix-react-sdk/pull/6700/commits/c6400c8db34e2384ce14ec461c04c99d3c8f3852)

Reply:
huh, I'm clearly losing my mind. I thought `DateUtils` was a component 🤦‍♂️